lib/atum/core/errors.rb

```diff
@@ -5,14 +5,34 @@ class SchemaError < StandardError; end
     class ResponseError < StandardError; end
 
     class ApiError < StandardError
-      attr_reader :error
+      attr_reader :request, :response
 
-      def initialize(error)
+      def initialize(request: request, response: response)
+        @request = request
+        @response = response
         @error = error
-        if @error.key?('documentation_url')
-          super("#{@error['message']}, see #{@error['documentation_url']}")
+      end
+
+      def error
+        @error ||=
+          if response.json?
+            response.body['error']
```

Comment:
Good shout - let's put the `response.body` into the `"Unknown error"` text below.